### PR TITLE
feat: APPS-1964 Create listing page for Collections

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-1064](https://jira.library.ucla.edu/browse/APPS-1964)

https://deploy-preview-517--uclalibrary-test.netlify.app/collections

### CODE REVIEW

File: pages/collections/index.vue [hunk 1]
```diff
@@ -1,71 +1,166 @@
...
+        <section-wrapper>
+            <divider-way-finder class="divider divider-way-finder" />
         </section-wrapper>
 
         <section-wrapper>
-            <section-staff-article-list
-                :items="parsedCollectionsList"
-                section-title="All Collections"
+            <section-cards-with-illustrations
+                class="section"
```
Comment (dianneweinthal) [inline pages/collections/index.vue:16]: can remove this line

File: pages/collections/index.vue [hunk 2]
```diff
@@ -1,71 +1,166 @@
...
         },
-    },
+    }
 }
 </script>
 
 <style lang="scss" scoped>
 .page-collections {
-    padding-left: 50px;
+    .section {
```
Comment (dianneweinthal) [inline pages/collections/index.vue:148]: can remove lines 148-161

File: pages/collections/index.vue [hunk 3]
```diff
@@ -1,71 +1,166 @@
...
+                    <button-more text="Browse Collections" />
+                </nuxt-link>
+            </section-wrapper>
+        </section-wrapper>
+
+        <section-wrapper>
+            <divider-way-finder class="divider divider-way-finder" />
+        </section-wrapper>
+
+        <section-teaser-card
```
Comment (dianneweinthal) [inline pages/collections/index.vue:58]: add section wrapper around cards with section-title "Collections News"

File: pages/collections/index.vue (hunk 2 above)
Comment (dianneweinthal) [reply pages/collections/index.vue:148]: add 
`.banner-text {
margin-top: var(--space-m);
}`

File: pages/collections/index.vue [hunk 4]
```diff
@@ -1,71 +1,183 @@
...
-import COLLECTIONS_LIST from "~/gql/queries/CollectionsList"
+import COLLECTIONS_LIST from "~/gql/queries/CollectionsList.gql"
 
 export default {
-    async asyncData({ $graphql, route }) {
-        // console.log("route: " + route.path)
-        const data = await $graphql.default.request(COLLECTIONS_LIST, {})
-        // console.log("data:" + data)
+    data() {
         return {
```
Comment (aprigge) [inline pages/collections/index.vue:89]: @matheusgomes062 is this needed?

**Pull request merged**